talent/onboarding/onboarding.md

```diff
@@ -0,0 +1,68 @@
+
+## Onboarding Process
+
+- This document describes the onboarding process for a new source{d} member.
+
+### Guide 
+
+- Read the source{d} [guide](https://github.com/src-d/guide) which contains all the information necessary from the moment you accept an offer. 
+
+### Hardware
+
+- A few days before you start working with us, you choose the hardware you would like to use from on the [list](https://github.com/src-d/guide/blob/master/general/available_hardware.md) with the hardware we provide. 
```

Comment:
Considering visual-impired people and screen readers, we're trying to avoid [surprise-links](https://www.google.es/search?q=cats&tbm=isch) in benefit of [links with auto descriptive titles](http://accessibility.oregonstate.edu//descriptivelinks).
(history: [@ricardobaeta proposed descriptive links here](https://github.com/src-d/guide/pull/84#discussion_r160802043))

For example, in this case, it could be:
> Here is a [list with the hardware we provide](https://github.com/src-d/guide/blob/master/communication/slack_channels.md).